bin/resty

```diff
@@ -213,6 +222,7 @@ for my $var (sort keys %ENV) {
 
 my $main_include_directives = resolve_includes('main', @main_includes);
 my $http_include_directives = resolve_includes('http', @http_includes);
+my $lua_shared_dicts = format_directives('lua_shared_dict', @shdicts);
```

Comment:
I think it's better to check the syntax of the shdict option values here since error messages from nginx might not be very friendly and helpful :)